server/web/account/__init__.py

```diff
@@ -0,0 +1,84 @@
+import datetime
```

Comment:
How do we actually navigate to these pages? There is no index route from which we can manage our account just the pages to do some of the management features. Navbar should have link to the index page.